WireMock/wwwroot/scripts/MatcherUI/PathMatcherUI.ts

```
@@ -47,8 +47,14 @@ export default class PathMatcherUI extends MatcherUI {
                 try {
                     let rawMapContent = JSON.parse(rawMap.textContent!);
                     rawMapContent.Request.Path.Matchers.splice(i, 1);
-                    rawMap.textContent = JSON.stringify(rawMapContent, null, 1);
-                    matcherRow.remove();
+                    if (rawMapContent.Request.Path.Matchers.length > 0) {
+                        rawMap.textContent = JSON.stringify(rawMapContent, null, 1);
+                        matcherRow.remove();
+                    }
+                    else {
+                        this.showErrorModal("You cannot remove the last matcher - " +
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)